### PR TITLE
fix: ELECTRON-1227 (Add translations for permissions dialog)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Add translations for permissions dialog
[ELECTRON-1227](https://perzoinc.atlassian.net/browse/ELECTRON-1227)

## Screenshots
- French
Screenshot 2019-07-04 at 2 12 28 PM
- Japanese
Screenshot 2019-07-04 at 2 11 16 PM

### CODE REVIEW

File: src/locale/en-US.json [hunk 1]
```diff
@@ -110,7 +110,12 @@
...
-    "Full Screen": "Full Screen"
+    "Your administrator has disabled Full Screen. Please contact your admin for help": "Your administrator has disabled Full Screen. Please contact your admin for help",
+    "Your administrator has disabled notifications. Please contact your admin for help": "Your administrator has disabled notifications. Please contact your admin for help",
+    "Your administrator has disabled Opening External App. Please contact your admin for help": "Your administrator has disabled Opening External App. Please contact your admin for help",
+    "Your administrator has disabled sharing your screen. Please contact your admin for help": "Your administrator has disabled sharing your screen. Please contact your admin for help",
+    "Your administrator has disabled sharing your camera, microphone, and speakers. Please contact your admin for help": "Your administrator has disabled sharing your camera, microphone, and speakers. Please contact your admin for help",
+    "Your administrator has disabled sharing your location. Please contact your admin for help": "Your administrator has disabled sharing your location. Please contact your admin for help"
```
Comment (keerthi16) [inline src/locale/en-US.json]: There are some missing strings

Comment (VishwasShashidhar) [inline src/locale/en-US.json]: we have 6 strings in this while we have 8 in `en.json`. Can you look into it?

File: src/locale/en-US.json [hunk 2]
```diff
@@ -110,7 +110,14 @@
...
+    "Your administrator has disabled MIDI Sysex. Please contact your admin for help": "Your administrator has disabled MIDI Sysex. Please contact your admin for help",
+    "Your administrator has disabled Opening External App. Please contact your admin for help": "Your administrator has disabled Opening External App. Please contact your admin for help",
+    "Your administrator has disabled Pointer Lock. Please contact your admin for help": "Your administrator has disabled Pointer Lock. Please contact your admin for help",
+    "Your administrator has disabled sharing your screen. Please contact your admin for help": "Your administrator has disabled sharing your screen. Please contact your admin for help",
+    "Your administrator has disabled sharing your camera, microphone, and speakers. Please contact your admin for help": "Your administrator has disabled sharing your camera, microphone, and speakers. Please contact your admin for help",
+    "Your administrator has disabled sharing your location. Please contact your admin for help": "Your administrator has disabled sharing your location. Please contact your admin for help"
```
Comment (keerthi16) [inline src/locale/en-US.json]: Not alphabetically ordered

**Comment by VishwasShashidhar** [comment]
@keerthi16 Can you approve this PR if it's good to go?

**Pull request merged**